playbooks/roles/backup-pruner/tasks/main.yml

```
@@ -29,9 +38,8 @@
   no_log: true
   with_items: "{{ BACKUP_PRUNER_TARSNAPPER_JOBS }}"
 
-# NOTE: Pip doesn't work with tarsnapper
 - name: install tarsnapper
-  command: easy_install "{{ BACKUP_PRUNER_TARSNAPPER_VERSION }}"
+  command: pip install tarsnapper
```

@farhaanbukhsh Can you fix the tarsnapper version here? So that when we provision a new server, it installs a known version instead of the latest one. It looks like it installed 0.5.0 on backup-pruner VM.
```suggestion
  command: pip install tarsnapper==0.5.0
```

---

Done :)